package.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "com.ivaschenko_dmitrii.statefului",
+    "version": "0.0.1",
+    "displayName": "Stateful UI",
+    "description": "Stateful UI",
+    "unity": "2022.3",
+    "author": {
+      "name": "Dmitrii Ivashchenko",
+      "email": "ivaschenko.dmitrii@gmail.com",
+      "url": "https://bit.ly/dmitry_ivashenko"
+    }
+  }
+  
```

Comment:
We already have the package.json inside Assets/Plugins/StatefulUI. 
You can install the library using UPM as described in README.
Please remove it.